CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # CHANGELOG
 
+## v2.0.1
+
+-- fixes #176 by surrounding comma-separated task list with `[]` in the Gruntfile `build` task definition
+
+## v2.0.0
+
+-- new makara/internationalization support
```

Comment:
Let's use markdown and `* item` rather than the badly-rendering `-- item`. Below in the file too.